frontend/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "edna-app",
+  "version": "0.1.0",
+  "private": true,
```

Comment:
We may make it public one day, right? At least in our dreams where this tool becomes very popular and the whole biotech community uses it…

Reply:
this flag is needed to not accidentally publish it as a npm package